Notify/Notify/Notify/ViewModels/RegistrationViewModel.cs

```diff
@@ -53,12 +64,12 @@ public bool IsFormValid
         private bool validateName()
         {
             bool isValid = !string.IsNullOrEmpty(Name) && Regex.IsMatch(Name, @"^[a-zA-Z ]+$");
+            NameBorderColor = isValid ? Color.SeaGreen : Color.Red;
```

Comment:
1. Use constants for valid color and invalid color.
2. Extract to a method, for being reused multiple times.